From 4d069862995f5b414fa3a06c4fd2b71e16defc26 Mon Sep 17 00:00:00 2001
From: Sorin Jianu <sorin@chromium.org>
Date: Thu, 30 Jun 2022 03:34:17 +0000
Subject: [PATCH 52/59] Disable IntegrationTest.SelfUpdateFromOldReal in M102.

Bug: 1340482
Change-Id: I1dc7d99a84880867c3afca5281edd80f0d41d4ed
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3732396
Reviewed-by: Joshua Pawlicki <waffles@chromium.org>
Commit-Queue: Sorin Jianu <sorin@chromium.org>
Cr-Commit-Position: refs/branch-heads/5005@{#1211}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 chrome/updater/test/integration_tests.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/updater/test/integration_tests.cc b/chrome/updater/test/integration_tests.cc
index d2d02dd04e7b..ca176f6016cc 100644
--- a/chrome/updater/test/integration_tests.cc
+++ b/chrome/updater/test/integration_tests.cc
@@ -620,7 +620,8 @@ TEST_F(IntegrationTest, UnregisterUnownedApp) {
 #if BUILDFLAG(CHROMIUM_BRANDING) || BUILDFLAG(GOOGLE_CHROME_BRANDING)
 // TODO(crbug.com/1268555): Even on Windows, component builds do not work.
 #if !defined(COMPONENT_BUILD)
-TEST_F(IntegrationTest, SelfUpdateFromOldReal) {
+// TODO(crbug.com/1340482) - disable the test on branches because it is flaky.
+TEST_F(IntegrationTest, DISABLED_SelfUpdateFromOldReal) {
   ScopedServer test_server(test_commands_);
 
   // TODO(crbug.com/1308856): Current versions of the updater do not send an
-- 
2.37.0

